integration/src/main/resources/suites/integration.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "FullIntegration",
+  "name": "integration",
```

Comment:
This renaming doesn't follow the convention used by other repos. Looking at workspace manager, a their approach is to put the test inside a directory, and use the directory name to determine which environment the test is written for. See https://github.com/DataBiosphere/terra-workspace-manager/tree/main/integration/src/main/resources/suites

Also should this be `dev` not `integration`? Or maybe `pull-request`?